src/scss/settings/_variables.scss

```diff
@@ -118,9 +118,9 @@ $zIndex: (
 
 // Path vars
 // ==========================================================================
-$img-path                  : '/assets/dist/img'; // Usage: background: url('#{$img-path}/img.jpg') 0 0 no-repeat;
-$icon-path                 : '#{$img-path}/icons';
-$font-path                 : '/assets/dist/fonts';
+$img-path                  : 'fozzie/dist/img' !default; // Usage: background: url('#{$img-path}/img.jpg') 0 0 no-repeat;
```

Comment:
The problem is that the package name is needed as a prefix to the directories here so that consumers of the package (global-component-library or app_globalweb) can find the `.svg`s in `/assets/dist/imported-assets/fozzie/...` when they run `gulp:css`. But since the `fozzie/` directory doesn't exist within this package, `gulp:css` errors here.

The only way I can find around this so far is to scrap the sorting of imported assets by directory [(removing pkg.name here)](https://github.com/justeat/gulp-build-fozzie/blob/master/tasks/copy.js#L105) but that would allow for potential filename conflicts if two packages export assets with the same path...